### PR TITLE
Installation instructions

### PR DESCRIPTION
**Pull request by musangowope**
- Enhance installation instructions with screenshots
- Demo 
https://www.loom.com/share/dce2d2c374864aaebc9667cabe348d98

resolves https://github.com/laws-africa/constitution-app-react/issues/69

### CODE REVIEW

File: src/pages/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -100,48 +135,97 @@ const LandingPage = () => {
...
+                        >
+                          bit.ly/concompass
+                        </a>
+                      </li>
+                      <li>
+                        Click on bottom floating banner with text: "Add
+                        Constitution Compass to Home screen" and prompts for
+                        install application with appear. If the banner is not
+                        visible click on the Toolbar icon on top right corner of
+                        the browser and select menu item, "Install App"
```
Comment (longhotsummer) [inline src/pages/LandingPage/LandingPage.tsx:223]:
```suggestion
                        Tap on "Add Constitution Compass to Home screen" and follow the prompts.
                        <br/><br/>
                        If the banner is not visible, tap on the Toolbar icon in top right corner, then tap "Install App".
```

**Comment by longhotsummer** [comment]
Nice.

I'm worried that the GIF is too fast. I certainly struggled to follow what was going on. We could also edit the movie file to highlight what needs to be tapped, since we can't see what the user is tapping.

@musangowope let's chat about it.

**Comment by longhotsummer** [comment]
@musangowope is it meant to look like this?

image

**Comment by musangowope** [comment]
> @musangowope is it meant to look like this?
> 
> image
yes. Do you not like the layout?

**Comment by musangowope** [comment]
@longhotsummer updated the layout as requested
https://www.loom.com/share/764192f8057949d98293417d7c32af95

**Review by longhotsummer — APPROVED** [review]
Huge improvement! 💪

**Pull request merged**